sw/device/silicon_creator/lib/manifest.h

```diff
@@ -334,7 +347,20 @@ inline epmp_region_t manifest_code_region_get(const manifest_t *manifest) {
 }
 
 /**
- * Gets the entry point of an image.
+ * Gets the executable execution region of the image.
+ *
+ * @param manifest A manifest.
+ * return Executable region of the image.
+ */
+inline epmp_region_t manifest_vma_code_region_get(const manifest_t *manifest) {
```

Comment:
I don't think we need this since we only need to know the slot we are unlocking for address translation configuration. Is that correct?

Also the region we unlock needs to be NAPOT aligned, the code section we use for ePMP may well not be.

Reply:
Sorry, I misunderstood what this was for.